jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -224,6 +224,18 @@ YUI.add('user-profile', function() {
       this.close();
     },
 
+    /**
+      Set to a new, blank canvas with no model.
+
+      @method resetModel
+    */
+    resetModel: function() {
+      this.props.switchModel();
```

Comment:
This is kind of confusing, I feel like it should take an argument to signify that it's being set to empty....like maybe
```JavaScript
  this.props.switchModel(null)
```